server/src/main/java/com/saralein/server/partial_content/RangeParser.java

```diff
@@ -0,0 +1,37 @@
+package com.saralein.server.partial_content;
+
+public class RangeParser {
+    public Range parse(String range, int fileLength) {
+        String modifiedRange = removeRangeUnit(range);
+        int hyphenIdx = modifiedRange.lastIndexOf("-");
```

Comment:
What is `Idx`?

Reply:
It's index.  Updated the variable name to `hyphenIndex`.